.github/CONTRIBUTING.md

```diff
@@ -4,9 +4,10 @@
 You should *always* use a virtual environment when developing software. Setup:
 
 ```bash
-python3 -m venv .env
+python -m venv .env
```

Comment:
Why are you dropping the 3? According to [PEP 0394](https://www.python.org/dev/peps/pep-0394/), `python` is not supposed to point at `python3` *still* (except in a virtual env), and you have to use Python 3 to use venv. (The dual-Python package you can install that does venv's is called `virtualenv` instead of `venv`.)

Modern systems are looking at just dropping `python` and providing `python2` and `python3` (such as RHEL 8).

Reply:
Actually, "Python3 -> Python" is recommended by Jim in https://github.com/scikit-hep/aghast/pull/31#discussion_r399652299. If you are sure using `python3` in command lines is a better manner. I will inform Jim to make some changes to Aghast, too.

Reply:
In Aghast, I asked Nino to drop the "3" in "python3" because that's particular to some OS distributions. I don't know how many systems do it one way and how many do it the other, but I would have thought it would be more common nowadays for plain old "python" to actually be Python 3. (It is on mine, for instance.)

These are instructions for a developer. I think one can just say, "Make sure to use Python 3" and however they do that on their system, they should do that.

Reply:
All Python 3 instances will always have `python3`. It is invalid according to PEP 0394 to have `python` be 3, but some systems have an "alternatives" system to change this, and a few modern systems just do this anyway, PEP or no PEP. In this case, we are running a Python 3 only builtin module (venv), not running setup.py.

In general, use python3 if the command only works on Python 3, python2 if it only works on 
Python 2, and python if it works in either.

If this was not aimed at developers, I'd probably point out there's a better but not builtin package called virtualenv.

Reply:
See https://linuxize.com/post/how-to-install-python-on-centos-8/

Reply:
I just realized the PEP was updated in 2019 to remove the requirements on `python` pointing at python2, and basically say anything goes. Good. Recommendation for us is still the same though. Jim, are you on a non LTS Ubuntu, perhaps, from 2019? Will be interesting if 20.04 has a Python 3 `python` command.

Reply:
Python2 does not have venv. It has virtualenv.

So I think this should be `python -m venv .env`.